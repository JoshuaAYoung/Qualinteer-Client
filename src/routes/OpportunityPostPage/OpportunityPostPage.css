.postInput {
  width: 100%;
  height: 3rem;
  font-size: 1.7rem;
  padding-left: 1rem;
}

.signupForm {
  margin-top: 6rem;
  margin-bottom: 6rem;
}

.postButton {
  margin-top: 2rem;
  margin-right: auto;
}

.editorContainer {
  margin: 2rem -2.5rem 0;
}

.postButtonContainer {
  text-align: center;
}

.dropdownButton {
  width: 4rem;
  height: 3rem;
  border-radius: 0;
  position: absolute;
  z-index: 1;
  right: 0;
}

.postSelect {
  border-right: 0;
  background-image: url('/assets/down-arrow.svg');
  background-repeat: no-repeat;
  background-size: 3rem;
  background-position: right;
  font-family: 'Montserrat', sans-serif;
  font-weight: 300;
  padding-top: 0.1em;
}

.loadingPost {
  width: 5.9rem;
  margin-top: 0.2rem;
}

.loadingPost > div {
  width: 1rem;
  height: 1rem;
}

@media only screen and (min-width: 520px) {
  .postInputs {
    padding-left: 10%;
    padding-right: 10%;
  }

  .oppPostBanner {
    padding-left: 10%;
    padding-right: 10%;
    text-align: center;
  }
}

@media only screen and (min-width: 768px) {
  .postInputs {
    padding-left: 10rem;
    padding-right: 10rem;
  }

  .oppPostBanner {
    padding-left: 0;
    padding-right: 0;
  }
}

@media only screen and (min-width: 1024px) {
  .postInputs {
    padding-right: 15rem;
    padding-left: 15rem;
  }
}
