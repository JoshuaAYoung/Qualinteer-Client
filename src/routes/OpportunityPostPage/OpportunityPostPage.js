import React from 'react';
import OpportunityPostForm from '../../components/OpportunityPostForm/OpportunityPostForm';
import './OpportunityPostPage.css';
import ScrollToTopOnMount from '../../utilities/ScrollToTopOnMount';

export default function OpportunityPostPage(props) {
  return (
    <div className="opportunityPostContainer">
      <ScrollToTopOnMount />
      <div className="oppPostBanner">
        <h1 className="pageHeader">Opportunity Post.</h1>
        <p className="pageInstructions">
          Fields are optional unless noted otherwise.
        </p>
      </div>
      <OpportunityPostForm history={props.history} />
    </div>
  );
}
