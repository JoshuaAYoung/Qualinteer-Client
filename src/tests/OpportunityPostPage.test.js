import React from 'react';
import ReactDOM from 'react-dom'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import OpportunityPostPage from '../routes/OpportunityPostPage/OpportunityPostPage';

describe('OpportunityPostPage Component', () => {

  it.skip('OpportunityPostPage Smoke Test', () => {
    const div = document.createElement('div')

    ReactDOM.render(<OpportunityPostPage />, div),
      ReactDOM.unmountComponentAtNode(div)
  })

  it('OpportunityPostPage Snapshot Test', () => {
    const wrapper = shallow(<OpportunityPostPage />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})