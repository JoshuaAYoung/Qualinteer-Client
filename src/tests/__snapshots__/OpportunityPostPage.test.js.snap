// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OpportunityPostPage Component OpportunityPostPage Snapshot Test 1`] = `
<div
  className="opportunityPostContainer"
>
  <ScrollToTopOnMount />
  <div
    className="oppPostBanner"
  >
    <h1
      className="pageHeader"
    >
      Opportunity Post.
    </h1>
    <p
      className="pageInstructions"
    >
      Fields are optional unless noted otherwise.
    </p>
  </div>
  <OpportunityPostForm />
</div>
`;
